pytorch_lightning/plugins/training_type/parallel.py

```diff
@@ -134,6 +134,9 @@ def block_backward_sync(self):
         else:
             yield None
 
+    def _config_model(self, model: "pl.LightningModule", device_ids: str, **ddp_kwarg) -> None:
+        self._model = DistributedDataParallel(model, device_ids=device_ids, **ddp_kwarg)
+
     def teardown(self) -> None:
```

Comment:
we shouldn't introduce DDP into the parallel plugin - rather we should do the opposite, and ensure parallel plugin stays as generic as possible. DDP is only one implementation, but other use cases may extend implement strategies using the parallel plugin very differently

Reply:
Got you! That make sense, will update to the other way